src/SmartComponents/BaselinesPage/EditBaseline/redux/reducers.js

```diff
@@ -93,6 +92,36 @@ export function editBaselineReducer(state = initialState, action) {
                 isCategory: action.payload.isCategory,
                 isSubFact: action.payload.isSubFact
             };
+        case `${types.SELECT_FACT}`: {
+            newEditBaselineTableData = [ ...state.editBaselineTableData ];
+            let isAllSelected = true;
+
+            newEditBaselineTableData.map(row => {
+                if (action.payload.ids.includes(row[0])) {
```

Comment:
What are `row[0]` and `row[2]` ? Casting them to variables at the top of this loop would help in understanding what is happening here,e.g.:

```
let id = row[0]
let ?? = row[2]
```